core/src/EditorFieldType/EditorFieldType.less

```diff
@@ -5,14 +5,14 @@
     display: flex;
     margin-bottom: 3px;
     align-items: flex-end;
-    font-size: @labelTextSize;
+    font-size: @FieldLabelSize;
 
     & span:first-child {
       flex: 1;
     }
 
     .EditorSelection {
-      margin-left: 10px;
+      margin-left: auto;
```

Comment:
Are you confident in this change? Going from a specific static value to an auto one seems like it could have unintended consequence.

Reply:
It aligns it to the right, by filling in the left, tested in manager app.